docs/design/datacontracts/data_descriptor.md

```diff
@@ -176,3 +176,89 @@ The indirection array is not part of the data descriptor spec.  It is expected t
 contract descriptor will include it. (The data contract descriptor must contain: the data
 descriptor, the set of compatible algorithmic contracts, the aux array of globals).
 
+## Example
+
+This is an example of a baseline descriptor for a 64-bit architecture. Suppose it has the name `"example-64"`
+
+```jsonc
+{
+  "version": 0,
+  "types": [
+    {
+      "name": "GCHandle",
+      "size": 8,
+      "fields": [
+        { "name": "Value", "type": "pointer", "offset": 0 }
+      ]
+    },
+    {
+      "name": "Thread",
+      "size": "indeterminate",
+      "fields": [
+        { "name": "ThreadId", "type": "uint32", "offset": "unknown" },
+        { "name": "Next", "type": "pointer" }, // offset "unknown" is implied
+        { "name": "ThreadState", "type": "uint32" }
+      ]
+    },
+    {
+      "name": "ThreadStore",
+      "fields": [
+        { "name": "ThreadCount", "type": "int32" },
+        { "name": "ThreadList", "type": "pointer" }
+      ]
+    }
+  ],
+  "globals": [
+    { "name": "FEATURE_EH_FUNCLETS", "type": "uint8", "value": "0" }, // baseline defaults value to 0
+    { "name": "s_pThreadStore", "type": "pointer" } // no baseline value
+  ]
+}
+```
+
+The following is an example of an in-memory descriptor that references the above baseline:
+
+```jsonc
+{
+  "version": "0",
+  "baseline": "example-64",
+  "types": [
+    {
+      "name": "Thread",
+      "fields": [
```

Comment:
We may want to use more compact style for key/value pairs for the in-memory descriptor at least, the json parser can support both less verbose and more verbose variants:

```
"types": [
    "Thread" : { "ThreadId" : 32, "ThreadState" : 0, "Next" : 128 }`,
...
```

I know the right way to serialize key/value pairs in JSON is one of the FAQs...

Reply:
Added a "compact" JSON variant to the spec